t/005-apicast-oauth.t

```diff
@@ -361,3 +362,38 @@ GET /?access_token=foobar
 --- error_code: 200
 --- response_body
 yay, upstream
+
+=== TEST 12: calling /authorize with state returns same value back on redirect_uri
```

Comment:
Well, we might want to add some additional ones sending the state parameter also, but since it's actually optional (but recommended) I think it makes sense to have separate test for it i.e test where client doesn't send state parameter vs tests where client sends state parameter